### PR TITLE
Release 1.5.1 commit

### PR DESCRIPTION
**Pull request by laurent-indermuehle**
Release 1.5.1 commit

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,13 +5,34 @@ Community MySQL Collection Release Notes
...
+
 v1.5.0
-=======
+======
 
 Release Summary
 ---------------
 
-This is a minor release of the ``community.mysql`` collection.
+This is bugfix release of the ``community.mysql`` collection.
```
Comment (Andersson007) [inline CHANGELOG.rst]:
```suggestion
This is bugfix release of the ``community.mysql`` collection.
```
i think it was a minor release (there were two ret values needed for CI added)

File: changelogs/changelog.yaml [hunk 2]
```diff
@@ -409,4 +409,30 @@ releases:
...
+      - Integration tests - Add more versions of MariaDB
+      - Integration tests - Add tools to test locally the same as on GHA by using
+        same containers and virtualenv. Custom test containers are published in ghcr.io
+        by this repo's workflows. MySQL/MariaDB are official Docker Hub images.
+      - Integration tests - Carefully verify every component of the tests in the new
+        target 'setup_controller' to ensure expected versions are correct Python,
+        Ansible, connector and MySQL/MariaDB.
+      - Integration tests - New name for many tasks to makes it easier to find failing
+        tests. Rename duplicates. Add name for tasks which doesn't had one, refactor
+        some tests files to better group tests by subject, ...
```
Comment (Andersson007) [inline changelogs/changelog.yaml]:
```suggestion
      minor_changes:
      - Integration tests - Add more versions of MariaDB
      - Integration tests - Add tools to test locally the same as on GHA by using
        same containers and virtualenv. Custom test containers are published in ghcr.io
        by this repo's workflows. MySQL/MariaDB are official Docker Hub images.
      - Integration tests - Carefully verify every component of the tests in the new
        target 'setup_controller' to ensure expected versions are correct Python,
        Ansible, connector and MySQL/MariaDB.
      - Integration tests - New name for many tasks to makes it easier to find failing
        tests. Rename duplicates. Add name for tasks which doesn't had one, refactor
        some tests files to better group tests by subject, ...
```
i did miss it anyhow. If it doesn't touch module interfaces for users i think we should remove these fragments?

Comment (laurent-indermuehle) [reply changelogs/changelog.yaml]: You mean, use "trivial" instead?

**Comment by laurent-indermuehle** [comment]
@Andersson007 I pushed 2 commits that I hope will fix the changelog. Is it good now?

File: changelogs/changelog.yaml (hunk 2 above)
Comment (Andersson007) [reply changelogs/changelog.yaml]: yes, you can but it's not needed in this repo, as trivial will be just ignored

Comment (Andersson007) [reply changelogs/changelog.yaml]: you can just remove the fragments before releasing. I'm ok with keeping it as is though. Ideally we shouldn't add any fragments for ci, docs, etc prs that don't touch code

**Review by Andersson007 — APPROVED** [review]
Thanks!

**Pull request merged**